Upstream: no
Issue: https://github.com/mikefarah/yq/issues/2822
Description: Test fails because it defaults to `yaml` for unknown extensions
diff --git a/test/format_test.go b/test/format_test.go
index ffa41ee..4ad4dae 100644
--- a/test/format_test.go
+++ b/test/format_test.go
@@ -17,7 +17,7 @@ func TestFormatStringFromFilename(t *testing.T) {
 		{"FILE.JSON", "json"},
 		{"file.properties", "properties"},
 		{"file.xml", "xml"},
-		{"file.unknown", "unknown"},
+		{"file.unknown", "yaml"},
 
 		// filenames without extensions
 		{"file", "yaml"},
